dp

```diff
@@ -34,11 +36,15 @@ EOF
 
 # Default settings
 PORT=22
-BASE_DIR="\$HOME/app"
+BASE_DIR="\$HOME/dp-apps"
+APP_NAME="app"
```

Comment:
Well, in the past we were only specific to one directory, `BASE_DIR`, but now we launch `svscan` to monitor `BASE_DIR` for any folders that are children of it an contain a `run` script, e.g., `*/run`. Any new folders created or deleted are checked by `svscan` to launch or destroy services.

It seems valuable to make it clear that we rely on this directory structure and that you need to be careful what directories you now create as children of `BASE_DIR`. Hence why I also changed the default, since monitoring `HOME` with svscan is fairly general, while monitoring `HOME/dp-apps` less so. I know to be careful what I create there.